cmd/git-sync/main.go

```diff
@@ -69,6 +71,22 @@ var flSSH = flag.Bool("ssh", envBool("GIT_SYNC_SSH", false),
 var flSSHKnownHosts = flag.Bool("ssh-known-hosts", envBool("GIT_KNOWN_HOSTS", true),
 	"enable SSH known_hosts verification")
 
+var webhooks = flag.String("webhooks", envString("WEBHOOKS", "[]"),
+	"the JSON formatted array of webhooks to be send a request to when git synced")
+var webhookTimeout = flag.Int("webhook-timeout", envInt("WEBHOOK_TIMEOUT", 60), 
+	"the integer number in seconds to set timeout for webhook call")	
+
+// WebHook structure
+type WebHook struct {
+	URL     string `json:"url"`
```

Comment:
Document these fields, please.  Does the URL include a scheme?  Does it support HTTPS?  What are valid values for method?  What does "Success" mean?